chainspecs/devSpec.json

```diff
@@ -11,6 +11,9 @@
     "tokenSymbol": "CTC"
   },
   "consensusEngine": null,
+  "grandpaInitialAuthorities": [
+    "5HHSXsD5zaMb9o4H7JzxUiAYpjArE4CANKbUBiFRVEwn6WPF"
```

Comment:
it would be nice to add a comment which account is that in case we need to be able to find it later.

I don't see this on my local node.

Reply:
It's the account ID of the first grandpa authority on devnet. For devnet, for instance, you can get it programmatically via something like

```typescript
const api = await ApiPromise.create({provider: new WsProvider("wss://rpc.devnet.creditcoin.network/ws")});
// find the block number where the PoS switch occurred
const switchBlock = await api.query.posSwitch.switchBlockNumber();
if (switchBlock.isNone) {
  throw new Error("PoS switch block not found");
}
// find the block hash associated with that block number
const blockHash = await api.rpc.chain.getBlockHash(switchBlock.unwrap());
// we want to query the API as of that block
const apiAtSwitch = await api.at(blockHash);
// get the grandpa authorities
const grandpaAuthoritiesRaw = await apiAtSwitch.call.grandpaApi.grandpaAuthorities();
// they're of the form [authorityId, weight], but we just want the authority IDs
const grandpaAuthorities = grandpaAuthoritiesRaw.map(([id, _weight]) =>
  id.toString()
);
console.log(`Initial authorities: ${grandpaAuthorities}`);
```

but the idea for mainnet is that this initial authority set would be determined ahead of time.

The actual account that acts as a grandpa authority is usually one created by calling `rotateKeys` (so you normally wouldn't save the private key aside from it being in the validator node's keystore)